From 43f90e26e6803af5377fd68679b69b067910c472 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Fri, 9 Jun 2023 16:06:44 -0400
Subject: Remove single quotes from filenames

---
 src/main.cpp | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/main.cpp b/src/main.cpp
index c89e86d..5cb4998 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -371,11 +371,11 @@ invokeLinker(const Parameters &params,
 
     string lwlinkCmdLine = params.lwlinkPath
                            + " --format=" + lwlinkFormat
-                           + " --output='" + outputFilename
-                           + "' --script='" + linkScriptFilename
-                           + "' --map='" + mapFilename + "'";
+                           + " --output=" + outputFilename
+                           + " --script=" + linkScriptFilename
+                           + " --map=" + mapFilename + "";
     for (vector<string>::const_iterator it = params.libDirs.begin(); it != params.libDirs.end(); ++it)
-        lwlinkCmdLine += " -L'" + *it + "'";
+        lwlinkCmdLine += " -L" + *it + "";
 
     lwlinkCmdLine += " -L" + params.cmocfloatlibdir;
     lwlinkCmdLine += " -lcmoc-crt-" + string(targetKW);
@@ -395,11 +395,11 @@ invokeLinker(const Parameters &params,
 
     for (vector<string>::const_iterator it = objectFilenames.begin();
                                        it != objectFilenames.end(); ++it)
-        lwlinkCmdLine += " '" + params.useIntDir(*it) + ".o'";
+        lwlinkCmdLine += " " + params.useIntDir(*it) + ".o";
 
     for (vector<string>::const_iterator it = libraryFilenames.begin();
                                        it != libraryFilenames.end(); ++it)
-        lwlinkCmdLine += " '" + *it + "'";
+        lwlinkCmdLine += " " + *it + "";
 
     if (params.verbose)
         cout << "Linker command: " << lwlinkCmdLine << endl;
-- 
