From 554f0188bd72ea548e1d1026b550c4d1adc98e70 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Fri, 9 Jun 2023 17:03:28 -0400
Subject: Remove single quotes in Parameters.cpp

---
 src/Parameters.cpp | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/Parameters.cpp b/src/Parameters.cpp
index e4ec79f..4a5d83d 100644
--- a/src/Parameters.cpp
+++ b/src/Parameters.cpp
@@ -246,7 +246,7 @@ Parameters::compileCFile(const string &inputFilename,
     stringstream cppCommand;
     cppCommand << "cpp -xc++ -U__cplusplus";  // -xc++ makes sure cpp accepts C++-style comments
     for (list<string>::const_iterator it = includeDirList.begin(); it != includeDirList.end(); ++it)
-        cppCommand << " -I'" << *it << "'";
+        cppCommand << " -I" << *it << "";
     cppCommand << " -D_CMOC_VERSION_=" << getVersionInteger();
     cppCommand << " -D" << targetPreprocId << "=1";
     if (!relocatabilitySupported)
@@ -256,7 +256,7 @@ Parameters::compileCFile(const string &inputFilename,
     cppCommand << " -U__GNUC__ -nostdinc -undef";
 
     for (list<string>::const_iterator it = defines.begin(); it != defines.end(); ++it)
-        cppCommand << " -D'" << *it << "'";
+        cppCommand << " -D" << *it << "";
 
     cppCommand << " " << inputFilename;  // must be last argument, for portability
 
@@ -484,9 +484,9 @@ Parameters::invokeAssembler(const string &inputFilename,
     string lwasmCmdLine = lwasmPath
                           + " -fobj --pragma=forwardrefmax"
                           + " -D" + targetPreprocId
-                          + " --output='" + objectFilename + "'"
-                          + (intermediateFilesKept ? " --list='" + lstFilename + "'" : "")
-                          + " '" + inputFilename + "'";
+                          + " --output=" + objectFilename + ""
+                          + (intermediateFilesKept ? " --list=" + lstFilename + "" : "")
+                          + " " + inputFilename + "";
     if (verbose)
         cout << "Assembler command: " << lwasmCmdLine << endl;
 
-- 
