From 00c828ec267bd6520830388dacdab8d0744ac522 Mon Sep 17 00:00:00 2001
From: Tim S <tim.stahlhut@gmail.com>
Date: Tue, 12 Mar 2019 12:48:05 -0400
Subject: build_msys2-mingw: Change default NATIVE_SYSTEM_HEADER_DIR

---
 gcc/Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/Makefile.in b/gcc/Makefile.in
index f50ea07bf..f30ae83fc 100644
--- a/gcc/Makefile.in
+++ b/gcc/Makefile.in
@@ -377,7 +377,7 @@ PARTITION_H = $(srcdir)/../include/partition.h
 MD5_H	    = $(srcdir)/../include/md5.h
 
 # Default native SYSTEM_HEADER_DIR, to be overridden by targets.
-NATIVE_SYSTEM_HEADER_DIR = /usr/include
+NATIVE_SYSTEM_HEADER_DIR = /include
 # Default cross SYSTEM_HEADER_DIR, to be overridden by targets.
 CROSS_SYSTEM_HEADER_DIR = @CROSS_SYSTEM_HEADER_DIR@
 
-- 
