From 15d8476728cfcfda6475b081ad787f6e8c582437 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Mon, 15 May 2023 11:36:40 -0400
Subject: Mingw build fix

---
 src/snapshot.c | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/src/snapshot.c b/src/snapshot.c
index 27f074e6..ec4e02d6 100644
--- a/src/snapshot.c
+++ b/src/snapshot.c
@@ -91,8 +91,11 @@ static void do_retry_read_snapshot(void *sptr) {
 
 int read_snapshot(const char *filename) {
 #ifndef HAVE_WASM
-	if (read_v2_snapshot(filename) < 0 &&
-	    read_v1_snapshot(filename) < 0) {
+	if (read_v2_snapshot(filename) < 0 
+#ifdef WANT_V1_SNAPSHOTS
+		&& read_v1_snapshot(filename) < 0
+#endif
+	) {
 		LOG_WARN("Snapshot: read failed\n");
 		return -1;
 	}
-- 
2.40.1.windows.1

