From e24725a154db271fb589a647dba1e45d253723fa Mon Sep 17 00:00:00 2001
From: Tim S <tim.stahlhut@gmail.com>
Date: Tue, 12 Feb 2019 14:57:03 -0500
Subject: core: add GNUC_STDC_INLINE

---
 gcc/cp/cfns.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gcc/cp/cfns.h b/gcc/cp/cfns.h
index 62cdfab93aa..2fb9b4632b9 100644
--- a/gcc/cp/cfns.h
+++ b/gcc/cp/cfns.h
@@ -53,6 +53,9 @@ __inline
 static unsigned int hash (const char *, unsigned int);
 #ifdef __GNUC__
 __inline
+#ifdef __GNUC_STDC_INLINE__
+__attribute__ ((__gnu_inline__))
+#endif
 #endif
 const char * libc_name_p (const char *, unsigned int);
 /* maximum key range = 391, duplicates = 0 */
-- 
