From 182052504ef92a9135353cdd65c8bb99f4b09b5a Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Mon, 15 May 2023 14:53:49 -0400
Subject: Clean fix

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 642661f..f9781c6 100644
--- a/Makefile
+++ b/Makefile
@@ -25,5 +25,5 @@ dasmfw: $(OBJS)
 	$(CXX) $(CPPFLAGS) $(<) -c -o $@
 
 .PHONY clean:
-	rm $(OBJS)
+	rm -f $(OBJS)
 	# rm $(PROGS)
-- 
