From 2cc4cb41b53353bb767b7ba10d625696e19bbab9 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Fri, 28 Apr 2023 12:17:46 -0400
Subject: printf debug added to insn_indexed.c

---
 lwasm/insn_indexed.c | 25 +++++++++++++++++++++++++
 1 file changed, 25 insertions(+)

diff --git a/lwasm/insn_indexed.c b/lwasm/insn_indexed.c
index c1d4e3e..c34e331 100644
--- a/lwasm/insn_indexed.c
+++ b/lwasm/insn_indexed.c
@@ -734,6 +734,10 @@ void insn_resolve_indexed_aux(asmstate_t *as, line_t *l, int force, int elen)
 				pb = ((l -> pb & 0x03) << 5) | (v & 0x1F);
 			}
 			l -> pb = pb;
+			if (l -> insn == 163 && l -> pb == 140)
+			{
+				fprintf(stderr, "%s\n", "debugging in insn_rel.c near line 739");
+			}
 			return;
 		}
 	}
@@ -746,10 +750,18 @@ void insn_resolve_indexed_aux(asmstate_t *as, line_t *l, int force, int elen)
 		l -> lint = 2;
 		goto do16bit;
 	}
+	if (l -> insn == 163 && l -> pb == 140)
+	{
+		fprintf(stderr, "%s\n", "debugging in insn_rel.c near line 755");
+	}
 }
 
 RESOLVEFUNC(insn_resolve_indexed)
 {
+	if (l -> insn == 163 && l -> pb == 140)
+	{
+		fprintf(stderr, "%s\n", "debugging in insn_rel.c near line 763");
+	}
 	if (l -> lint == -1)
 		insn_resolve_indexed_aux(as, l, force, 0);
 	
@@ -760,6 +772,10 @@ RESOLVEFUNC(insn_resolve_indexed)
 		else
 			l -> len = OPLEN(instab[l -> insn].ops[0]) + l -> lint + 1;
 	}
+	if (l -> insn == 163 && l -> pb == 140)
+	{
+		fprintf(stderr, "%s\n", "debugging in insn_rel.c near line 777");
+	}
 }
 
 void insn_emit_indexed_aux(asmstate_t *as, line_t *l)
@@ -773,6 +789,7 @@ void insn_emit_indexed_aux(asmstate_t *as, line_t *l)
 		i = lw_expr_intval(e);
 		if (i < -128 || i > 127)
 		{
+			fprintf(stderr, "%s and i=%d pb=%d insn=%d\n", "debugging in insn_rel.c near line 784", i, l -> pb, l -> insn);
 			lwasm_register_error(as, l, E_BYTE_OVERFLOW);
 		}
 	}
@@ -829,5 +846,13 @@ void insn_emit_indexed_aux(asmstate_t *as, line_t *l)
 
 EMITFUNC(insn_emit_indexed)
 {
+	if (l -> insn == 163 && l -> pb == 140)
+	{
+		fprintf(stderr, "%s\n", "debugging in insn_rel.c near line 851");
+	}
 	insn_emit_indexed_aux(as, l);
+	if (l -> insn == 163 && l -> pb == 140)
+	{
+		fprintf(stderr, "%s\n", "debugging in insn_rel.c near line 856");
+	}
 }
-- 
