From ea0ad42c7cbd01df1bf65ba83c026dcd59ad7b15 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Tue, 9 Apr 2024 16:23:43 -0400
Subject: Edit lib/Makefile

---
 lib/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lib/Makefile b/lib/Makefile
index 120fbdd..2a46964 100644
--- a/lib/Makefile
+++ b/lib/Makefile
@@ -6,7 +6,7 @@ STRING = strlen.o strcmp.o strpbrk.o strtok.o strspn.o patmatch.o strucmp.o \
 MEM = mem.o memory.o memchr.o memccpy.o memcmp.o memset.o memcpy.o memchr.o \
 	memncmp.o
 SYS = sleep.o
-IO = printf.o pffinit.o pflinit.o pfldummy.o putc.o puts.o iob_data.o setbase.o fseek.o fopen.o
+IO = printf.o pflinit.o pfldummy.o putc.o puts.o iob_data.o setbase.o fseek.o fopen.o
 LOW_IO = access.o create.o open.o read.o write.o
 MISC = rpterr.o chcodes.o htoi.o htol.o rand.o prof.o
 MATH_SUPPORT = ccmult.o ccdiv.o ccmod.o atoi.o abs.o
-- 
2.44.0

